packages/utils/src/translation-helpers.ts

```diff
@@ -35,7 +36,12 @@ export function getTranslateableFields(
 
   // blocks
   traverse(blocks).forEach(function (el) {
-    // TODO: localized custom components inputs
+    if (this.key && el && el['@type'] === localizedType) {
```

Comment:
localized types can reference any time, not just `text`, we need to exclude that, maybe set a meta property on the element to flag the text inputs